public/index.html

```diff
@@ -6,30 +6,30 @@
     <meta name="theme-color" content="#000000" />
     <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0,viewport-fit=cover">
     <meta name="title" content="Genetic Password Generator (GPG)">
-    <meta name="description" content="Password Generator using Genetic Algorithm for syllable password">
-    <meta name="keywords" content="password, generator, genetic algorithm">
+    <meta name="description" content="Password Generator bahasa Indonesia menggunakan Genetic Algorithm untuk random generator dan kamus sesuai kbbi kata benda dan kata sifat">
```

Comment:
```suggestion
    <meta name="description" content="Password Generator bahasa Indonesia menggunakan Genetic Algorithm untuk random generator dan kamus sesuai KBBI kata benda dan kata sifat">
```

I think we can change `kbbi` to `KBBI` because it's an acronym (?)